### PR TITLE
feat: branching and release strategy

### PR DESCRIPTION
**Pull request by mbritense**
-

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -150,6 +150,7 @@
   * ZGW
     * [Documents](/using-valtimo/zgw/documents/zgw-documents.md)
 * [Extending Valtimo](extending-valtimo/extending-valtimo.md)
+  * [Branching and release strategy](extending-valtimo/branching-and-release-strategy.md)
```
Comment (ivo-ritense) [inline SUMMARY.md:153]: I don't think this is the right location. "Extending Valtimo" is about how you can use certain valtimo features that require code in your implementation to use. This is not about implementing valtimo, but about developing valtimo itself. I would suggest putting it on the same level as the contributing to valtimo page. Or maybe as a sub page of that.

File: extending-valtimo/branching-and-release-strategy.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+If the use case calls for it, feature toggles can be retained if a feature is finished, for example if the feature
+needs to be turned off by default.
+
+## Releasing
+
+Changes in `next-minor` in both `valtimo-frontend-libraries` and `valtimo-backend-libraries` are released as frequently
+as possible, but only when the branches are stable. There is a maximum of 4 weeks between releases.
+
+A release is done by tagging the commit that is to be released. After this, a short-lived release branch is created,
```
Comment (ivo-ritense) [inline extending-valtimo/branching-and-release-strategy.md:55]: I think this is incorrect. The tag is only created at the end, when the rc branch has been tested and an artifact is released in some repository.

File: extending-valtimo/branching-and-release-strategy.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+Both [valtimo-frontend-libraries](https://github.com/valtimo-platform/valtimo-frontend-libraries) and [valtimo-backend-libraries](https://github.com/valtimo-platform/valtimo-backend-libraries/)
+are mainly developed in one single branch: `next-minor`. As its name implies, all commits made into `next-minor` will be
+released publicly when the next minor version of the product is released.
+
+### `next-major`
+
+Breaking changes for the product are developed in the `next-major` branch. Only changes related to the breaking change
+are made here. Unrelated changes are still committed to `next-minor`.
+
+While in development, `next-minor` is frequently merged into `next-major`.
```
Comment (ThomasMinkeRitense) [inline extending-valtimo/branching-and-release-strategy.md:19]: Suggestion: while next-major is in development...

**Comment by ThomasMinkeRitense** [comment]
In general: I see 'branched of', 'branched off' and 'branched from', 'based on', 'based of' and 'based off'. Please pick one (or multiple) and change the others.

**Review by ThomasMinkeRitense — COMMENTED** [review]
In general: I see 'branched of', 'branched off' and 'branched from', 'based on', 'based of' and 'based off'. Please pick one (or multiple) and change the others.

File: SUMMARY.md [hunk 4]
```diff
@@ -8,8 +8,10 @@
     * [Documents](introduction/modules/document-introduction.md)
     * [Form flow](introduction/modules/form-flow-introduction.md)
     * [Plugins](introduction/modules/plugin-introduction.md)
-  * [How to contribute](CONTRIBUTING.md)
   * [Style guide for contributing](STYLE-GUIDE.md)
```
Comment (marijnritense) [inline SUMMARY.md:11]: This should move to the contributing folder as well then, right?

File: contributing/branching-and-release-strategy.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+The Valtimo product aims to deliver features to its user quickly. Contributions from outside the core
+development team are welcomed. To make this possible, the below branching and release strategy is used.
+
+## Branching
+
+### `next-minor`
+
+Both [valtimo-frontend-libraries](https://github.com/valtimo-platform/valtimo-frontend-libraries) and [valtimo-backend-libraries](https://github.com/valtimo-platform/valtimo-backend-libraries/)
+are mainly developed in one single branch: `next-minor`. As its name implies, all commits made into `next-minor` will be
+released publicly when the next minor version of the product is released.
```
Comment (marijnritense) [inline contributing/branching-and-release-strategy.md:12]:
```suggestion
included when the next minor version of the product is released.
```

Comment (marijnritense) [reply contributing/branching-and-release-strategy.md:12]: Maybe also mention that `next-minor` will also be merged into `next-major` when a minor is released?

File: contributing/branching-and-release-strategy.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+Once the breaking changes in `next-major` are finished and tested, `next-major` is merged into `next-minor`. This
+commit is tagged and released. Work then continues in the `next-minor` branch until a new breaking change needs to be
+made, and a new `next-major` branch is created from `next-minor`.
+
+### Back-end libraries
+
+Because of security reasons, back-end features are developed in feature branches, branched off from `next-minor`. 
+Feature branches are named `feature/*`. Parts of the feature are developed in branches prefixed with `story/`,
+which include a story number. For example: `story/12345-add-endpoint`. These story branches are merged into
```
Comment (marijnritense) [inline contributing/branching-and-release-strategy.md:29]: I wouldn't mention the story number here. For external contributions, that wouldn't work out as they do not have story numbers.

File: contributing/branching-and-release-strategy.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+the feature branch as soon as possible. In addition, `next-minor` is merged frequently into the feature branch.
+
+Bugfix branches follow the naming scheme: `bugfix/12345-fix-security`. They are either merged into `next-minor`,
```
Comment (marijnritense) [inline contributing/branching-and-release-strategy.md:32]: Wouldn't mention the story number here either

**Review by ivo-ritense — APPROVED** [review]

**Review by ivo-ritense — APPROVED** [review]

**Pull request merged**